css/main.css

```diff
@@ -1,22 +1,18 @@
 @import "card.css";
 
-body {
-  background: lightblue;
-}
+/* General styling that applies to all pages */
 
-nav {
-  padding-bottom: 14px;
+:root {
+  --accent-color: #54acdc;
 }
 
-nav a {
-  padding: 15px;
-  background: white;
-  border-top-left-radius: 20px;
-  border-top-right-radius: 20px;
+body {
+background: var(--accent-color);
+font-family: 'Inter', sans-serif;
 }
 
-nav a:hover:not(.active) {
-  background: grey;
+h1, h2, h3, h4, h5, h6 {
+font-family: 'Zen Dots', cursive;
```

Comment:
would it be worth specifying font fallbacks if Zen Dots doesn't load? I don't know much about fonts

Reply:
`cursive` would be the fallback here. (see https://www.w3schools.com/css/css_font_fallbacks.asp)

Though looking at the cursive fallback it doesn't look great. Changed it to use the `fantasy` fallback font instead lol;;;